diff -urN yafc-1.1.orig/cf/readline.m4 yafc-1.1/cf/readline.m4
--- yafc-1.1.orig/cf/readline.m4	2002-12-05 23:10:30.000000000 +0100
+++ yafc-1.1/cf/readline.m4	2004-08-01 18:18:51.000000000 +0200
@@ -20,7 +20,7 @@
 
 AC_DEFUN(YAFC_CHECK_READLINE,
 [
-  AC_SEARCH_LIBS(tgetent, [ncurses curses termcap], [])
+  AC_SEARCH_LIBS(tgetent, [tinfo ncurses curses termcap], [])
 
   AC_MSG_CHECKING([for readline >= 2.0])
 
