diff -ruN yafc-1.1.1./lib/gssapi.c yafc-1.1.1/lib/gssapi.c
--- yafc-1.1.1./lib/gssapi.c	2006-02-14 18:24:49.547385304 +0100
+++ yafc-1.1.1/lib/gssapi.c	2006-02-14 18:24:34.657648888 +0100
@@ -72,7 +72,7 @@
 static gss_OID_desc gss_c_nt_hostbased_service_oid_desc =
 {10, (void *)"\x2a\x86\x48\x86\xf7\x12" "\x01\x02\x01\x04"};
 
-static gss_OID GSS_C_NT_HOSTBASED_SERVICE = &gss_c_nt_hostbased_service_oid_desc;
+gss_OID GSS_C_NT_HOSTBASED_SERVICE = &gss_c_nt_hostbased_service_oid_desc;
 
 
 
