--- yafc-0.7.10/configure.in.orig	Fri May 31 09:41:21 2002
+++ yafc-0.7.10/configure.in	Mon Oct 21 17:47:50 2002
@@ -107,7 +107,7 @@
 
 AC_CHECK_FUNCS(getopt_long)
 if test "$ac_cv_func_getopt_long" != "yes"; then
-  LIBOBJS="$LIBOBJS getopt.o getopt1.o"
+  AC_LIBOBJ="$AC_LIBOBJ getopt.o getopt1.o"
 fi
 
 AC_CHECK_FUNC(regcomp,
@@ -137,7 +137,7 @@
 
 dnl AC_FUNC_FNMATCH
 dnl if test "$ac_cv_func_fnmatch_works" != "yes"; then
-dnl   LIBOBJS="$LIBOBJS fnmatch.o"
+dnl   AC_LIBOBJ="$AC_LIBOBJ fnmatch.o"
 dnl else
 dnl   AC_CHECK_HEADERS(fnmatch.h)
 dnl fi
@@ -180,7 +180,7 @@
 
 dnl     if test "$ac_cv_lib_ssl_SSL_connect" = "yes"; then
 dnl       AC_DEFINE(USE_SSL)
-dnl       LIBOBJS="$LIBOBJS security.o ssl.o"
+dnl       AC_LIBOBJ="$AC_LIBOBJ security.o ssl.o"
 dnl       AC_CHECK_HEADERS(openssl/ssl.h)
 dnl     else
 dnl       AC_WARN(SSL library not found, SSL support not enabled)
@@ -196,7 +196,7 @@
 if test "$mhe_with_krb4" = "yes" -o "$mhe_with_krb5" = yes; then
   AC_MSG_RESULT(yes)
 
-  LIBOBJS="$LIBOBJS security.o"
+  AC_LIBOBJ="$AC_LIBOBJ security.o"
 
   AC_CHECK_LIB(des, des_pcbc_encrypt)
 
@@ -218,7 +218,7 @@
             AC_DEFINE(KRB4)
             AC_CHECK_FUNCS(krb_get_our_ip_for_realm)
             AC_CHECK_HEADERS(krb.h)
-            LIBOBJS="$LIBOBJS krb4.o kauth.o"
+            AC_LIBOBJ="$AC_LIBOBJ krb4.o kauth.o"
           ])
         fi
 
@@ -298,7 +298,7 @@
               if test "$ac_cv_lib_gssapi_gss_init_sec_context" = "yes"; then
                 AC_DEFINE(KRB5)
                 AC_CHECK_HEADERS(krb5.h gssapi.h)
-                LIBOBJS="$LIBOBJS gssapi.o"
+                AC_LIBOBJ="$AC_LIBOBJ gssapi.o"
               else
                 found_all_krb5=0;
                 LIBS="$SAVELIBS"
@@ -312,7 +312,7 @@
                 AC_CHECK_HEADERS(krb5.h)
                 AC_CHECK_HEADERS(gssapi.h gssapi/gssapi.h)
                 AC_CHECK_HEADERS(gssapi/gssapi_generic.h gssapi/gssapi_krb5.h)
-                LIBOBJS="$LIBOBJS gssapi.o"
+                AC_LIBOBJ="$AC_LIBOBJ gssapi.o"
               else
                 found_all_krb5=0;
                 LIBS="$SAVELIBS"
